src/main/java/com/urswolfer/intellij/plugin/gerrit/util/UrlUtils.java

```diff
@@ -43,4 +45,11 @@ public static URI createUriFromGitConfigString(String gitConfigUrl) {
     public static String stripGitExtension(String url) {
         return url.replace(".git", ""); // some repositories end their name with ".git"
     }
+
+    public static String encodePatchSetDescription(String text) {
+        return URLEncoder.encode(text, StandardCharsets.UTF_8)
+            .replaceAll("\\.", "%2E")
+            .replaceAll("-", "%96")
```

Comment:
Have you found any documentation which chars need to be replaced? I would move the comment from the test up here, and add a link if possible.

Also note that `replaceAll` requires a regex - I do not think you are currently using a regex here?

Reply:
True, I was thinking about moving the comment as well. 
I will add the link to the comment: https://gerrit-review.googlesource.com/Documentation/user-upload.html#patch_set_description

Yes, you are right, it's not necessary to use RegEx, I'll change it to replace(...)